stdlib/2.7/socket.pyi

```diff
@@ -278,7 +278,7 @@ class socket:
     proto = 0
 
     def __init__(self, family: int = ..., type: int = ...,
-                 proto: int = ..., fileno: int = ...) -> None: ...
+                 proto: int = ..., fileno: Optional[int] = ...) -> None: ...
```

Comment:
from socket.py:
```
class _socketobject(object):
    ...
    def __init__(self, family=AF_INET, type=SOCK_STREAM, proto=0, _sock=None):
        if _sock is None:
            _sock = _realsocket(family, type, proto)
        self._sock = _sock
        ...
socket = SocketType = _socketobject
```
So it appears to take *an* argument.  In the code I'm writing I'm subclassing socket so it's weird if it takes different arguments.  I could make my subclass not take a fileno also so I guess that's a perfectly valid solution.